docs/resources/Channel.md

```diff
@@ -1068,7 +1069,7 @@ Files must be attached using a `multipart/form-data` body as described in [Uploa
 ###### JSON/Form Params
 
 > info
-> When creating a message, apps must provide a value for **at least one of** `content`, `embeds`, `sticker_ids`, `components`, or `files[n]`.
+> When creating a message, apps must provide a value for **at least one of** `content`, `embeds`, `sticker_ids`, `components`, `files[n]`, or `poll`.
```

Comment:
There is a similar info block for [Execute Webhook](https://discord.com/developers/docs/resources/webhook#execute-webhook) that hasn't been updated